scan.go

```diff
@@ -1307,6 +1307,10 @@ func stringToCm(token string) (e, m uint8, ok bool) {
 }
 
 func toAbsoluteName(name, origin string) (absolute string, ok bool) {
+	if name == "\n" {
```

Comment:
and move this to line 1322, the `@` is more important then this corner case and also happens more often